app.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const { getTopics } = require('./controllers/topics-controllers');
+
+const app = express();
+
+app.get(`/api/topics`, getTopics);
+
+app.use((err, req, res, next) => {
+  if (err.code) {
+    res.status(400).send({ msg: 'Bad request' });
+  } else {
+    next(err);
+  }
```

Comment:
You may want to test the actual error codes that come back (e.g. 22P02) and give a bespoke error message back to the user :)